libs/pages/elewa/home/src/lib/components/home-lander-section/home-lander-section.component.html

```diff
@@ -0,0 +1,8 @@
+<elewa-group-elewa-hero 
```

Comment:
You're supposed to reuse the dynamic hero component that's already created and pass the correct attributes as inputs.